### DISCUSSION TITLE
post/boids-on-gba

### DISCUSSION THREAD

**Discussion by giscus** [foopod/blog6#11] [General]
# post/boids-on-gba

https://jonoshields.com/post/boids-on-gba

**Comment by ClementineAccount** [comment]
Hi! 

I noticed that this blog post forgot to mention that the .gba file was created using [butano](https://github.com/GValiente/butano). I was interested in how it was exported. Luckily I manage to find it by looking at the makefile though! Still interesting topic though I remember [Sebastian Lague](https://www.youtube.com/watch?v=bqtqltqcQhw)
